model/registry/user.yaml

```diff
@@ -28,8 +28,16 @@ groups:
         type: string
         stability: experimental
         brief: >
-          Unique identifier of the user.
+          Unique identifier of an authenticated user.
         examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
+      - id: anonymous_id
+        type: string
+        stability: experimental
+        brief: >
+          Identifier for a user who interacts with a system. This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
```

Comment:
@trisch-me can you comment on this description? 
I can update it to include a condition like `when auth context is present` it should be treated as authenticated user id. something like that.

let's improve this description.

Reply:
I have updated original id to represent auth context